### PR TITLE
feat: support grafanalib

### PR DESCRIPTION
**Pull request by jacky-xbb**

### CODE REVIEW

File: config/grafana-template/grafanalib-emqx/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- Install Enterprise Dashboard
+
+```shell
+./emqx5.py > emqx5.json
+```
+
+- Install OpenSource Dashboard 
+
+```
+./emqx5.py -c os > emqx5.json
```
Comment (Rory-Z) [inline config/grafana-template/grafanalib-emqx/README.md:38]: `os` has ambiguity, maybe can use `opensource` or `ce(Community Edition)`

File: config/grafana-template/grafanalib-emqx/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+- Install OpenSource Dashboard 
+
+```
+./emqx5.py -c os > emqx5.json 
+```
+
+
+
+
```
Comment (Rory-Z) [inline config/grafana-template/grafanalib-emqx/README.md:43]: Please delete empty line

File: config/grafana-template/grafanalib-emqx/README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+pipenv shell
+```
+
+
+## Generate Dashboard JSON
+
+- Install Enterprise Dashboard
+
+```shell
+./emqx5.py > emqx5.json
```
Comment (Rory-Z) [inline config/grafana-template/grafanalib-emqx/README.md:32]: I think it's easy to confuse the enterprise version with the open-source version. Maybe we can also add `-c` just like the open-source version?

File: config/grafana-template/grafanalib-emqx/README.md (hunk 1 above)
Comment (Rory-Z) [reply config/grafana-template/grafanalib-emqx/README.md:38]: By the way, what is the mean for option `c`?

Comment (jacky-xbb) [reply config/grafana-template/grafanalib-emqx/README.md:38]: c is a shortcut for category

**Review by Rory-Z — APPROVED** [review]

**Pull request merged**